pyart/graph/radardisplay.py

```diff
@@ -98,10 +106,30 @@ def __init__(self, radar, shift=(0.0, 0.0)):
         else:
             self.origin = 'radar'
 
+        self.shift = shift
+        self._calculateLocalization(radar)
+
+        # datetime object describing first sweep time
+        times = radar.time['data'][0]
+        units = radar.time['units']
+        calendar = radar.time['calendar']
+        self.time_begin = netCDF4.num2date(times, units, calendar)
+
+        # sweep start and end indices
+        self.starts = radar.sweep_start_ray_index['data']
+        self.ends = radar.sweep_end_ray_index['data']
+
+        # list to hold plots, plotted fields and plotted colorbars
+        self.plots = []
+        self.plot_vars = []
+        self.cbs = []
+
+    def _calculateLocalization(self, radar):
```

Comment:
Use _calculate_localization for this function name.  Camel Case is not recommended for functions in Python (and is not use in Py-ART that I know of).